README.md

```diff
@@ -3,11 +3,31 @@
 [![CircleCI](https://img.shields.io/circleci/project/gliderlabs/glidergun/release.svg)](https://circleci.com/gh/gliderlabs/glidergun)
 [![IRC Channel](https://img.shields.io/badge/irc-%23gliderlabs-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#gliderlabs)
 
+glidergun allows writing write-once-run-anywhere cli tools using  [Go](https://golang.org) and [Bash](http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO.html).
 
-## Install
+# Install
 
-	$ curl https://dl.gliderlabs.com/gh/glidergun/latest/$(uname -sm|tr \  _).tgz \
-		| tar -zxC /usr/local/bin
+    $ curl https://dl.gliderlabs.com/gh/glidergun/latest/$(uname -sm|tr \  _).tgz \
+      | tar -zxC /usr/local/bin
+
+
+# How it works 
+
+The `gun`-command will act as a central point for accessing subcommands written in GO+Bash.
```

Comment:
Again, maybe not mention Go. Go is just used for the glidergun binary. Try dropping the - in `gun`-command